lib/open_weather/endpoints/stations.rb

```diff
@@ -30,6 +30,19 @@ def delete_station(id)
         nil
       end
 
+      def create_measurements(measurements, options = {})
+        post('measurements', options.merge(body: measurements))
+        nil
+      end
+
+      def get_measurements(options)
+        required_keys = %i[station_id type limit from to]
+        missing_keys = required_keys - options.keys
+        raise ArgumentError, "Missing params: #{missing_keys.join(', ')}" unless missing_keys.empty?
+
+        get('measurements', options).map { |m_hash| OpenWeather::Models::Stations::Measurement.new(m_hash) }
```

Comment:
m_hash is a weird name, just make it `|m|` or `|h|`, since it's a temporary var

Reply:
btw, for a future PR you might want to consider integrating something like [dry-types](https://github.com/dry-rb/dry-types) where there's an explicit way to define required arguments and check for them